src/jit/simd.cpp

```diff
@@ -2622,6 +2628,46 @@ GenTreePtr Compiler::impSIMDIntrinsic(OPCODE                opcode,
         }
         break;
 
+        case SIMDIntrinsicNarrow:
+        {
+            assert(!instMethod);
+            op2 = impSIMDPopStack(simdType);
+            op1 = impSIMDPopStack(simdType);
+
+            simdTree = gtNewSIMDNode(simdType, op1, op2, simdIntrinsicID, baseType, size);
+            retVal   = simdTree;
```

Comment:
It would be a good idea to add a comment here explaining what op1 and op2 is so that one doesn't have to look at managed API to understand this.